docker/test/performance-comparison/compare.sh

```diff
@@ -537,9 +537,20 @@ unset IFS
 # all nodes.
 numactl --show
 numactl --cpunodebind=all --membind=all numactl --show
-# Use less jobs to avoid OOM. Some queries can consume 8+ GB of memory.
-jobs_count=$(($(grep -c ^processor /proc/cpuinfo) / 4))
-numactl --cpunodebind=all --membind=all parallel --jobs  $jobs_count --joblog analyze/parallel-log.txt --null < analyze/commands.txt 2>> analyze/errors.log
+
+# Notes for parallel:
+#
+# Some queries can consume 8+ GB of memory, so it worth to limit amount of jobs
+# that can be run in parallel.
+#
+# --memfree:
+#
+#   will kill jobs, which is not good (and retried until --retries exceeded)
+#
+# --memsuspend:
+#
+#   If the available memory falls below 2 * size, GNU parallel will suspend some of the running jobs.
+numactl --cpunodebind=all --membind=all parallel -v --jobs --joblog analyze/parallel-log.txt --memsuspend 15G --null < analyze/commands.txt 2>> analyze/errors.log
```

Comment:
are you sure that argument for `--jobs` might be absent? for me it doesn't work

Reply:
Let's see, here is how I tested:

```sh
$ time yes 'sleep 1' | head -n10 | parallel --jobs 1

real    0m10.959s
user    0m0.094s
sys     0m0.065s

$ time yes 'sleep 1' | head -n10 | parallel

real    0m1.133s
user    0m0.135s
sys     0m0.100s
```

And about `memsuspend` thing in particular:

````sh
$ timeout -v 3s parallel --memsuspend 1G <<<'echo foo'
foo
$ timeout -v 3s parallel --memsuspend 1T <<<'echo bar' # I don't have 1T of RAM
timeout: sending signal TERM to command ‘parallel’
```

@nickitat maybe you can share you oneliners so that I can take a look?

Reply:
Oh, I haven't removed `--jobs`, fixed.

Reply:
I mean it should be `--jobs N`, but now it is just `--jobs`

Reply:
Yep, I've just removed `--jobs` option, defaults looks OK:

> 0 means as many as possible (this can take a while to determine). Default is 100% which will run **one job per CPU thread on each machine.**

Reply:
So parallel from ubuntu:20 is too old, and does not support this option. Any objections for using ubuntu:22 for performance tests?

Reply:
no